httpclient/src/main/java/org/apache/http/impl/execchain/ResponseEntityProxy.java

```diff
@@ -173,4 +177,20 @@ public String toString() {
         return sb.toString();
     }
 
+    public List<Header> getTrailers() {
```

Comment:
@Cole-Greer Do you mind making this method package private, so it does not show up in the public API, but still could be called reflectively, when need be?

Reply:
@michael-o Not really. Considering how lax we used to be with API visible additions in the past I am being extreme here. Shall we keep it public?

Reply:
I would say so

Reply:
Thanks for always being quick and friendly with your responses. I look forward to the next release.